edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Docker/CombinedEdgeletConfigProvider.cs

```diff
@@ -139,5 +143,44 @@ void MountSockets(IModule module, CreateContainerParameters createOptions)
                 SetMountOptions(createOptions, managementUri);
             }
         }
+
+        void SetCapabilities(CreateContainerParameters createOptions)
+        {
+            // These capabilities are provided by default and can be used to gain root access:
+            // https://labs.f-secure.com/blog/helping-root-out-of-the-container/
+            HashSet<string> capabilitiesToRemove = new HashSet<string> { "CAP_CHOWN", "CAP_SETUID" };
+
+            // If customer manually adds the capabilites, don't drop them.
```

Comment:
Typo: capabilities

Also, where do these capabilities come from if not the customer? Who else can set them?